### PR TITLE
changed

### PR DESCRIPTION
**Pull request by AlexandraKovaleva**

### CODE REVIEW

File: hello_world.py [hunk 1]
```diff
@@ -1 +1 @@
-print("Hello, world!")
\ No newline at end of file
+print("Hello, Alex!")
```
Comment (alexandra-kovaleva) [inline hello_world.py]: Do you think, that Alex is equal to world?

Comment (AlexandraKovaleva) [reply hello_world.py]: yep

Comment (alexandra-kovaleva) [reply hello_world.py]:
```Suggestion:
print("Hello, everyone!")
```

Comment (alexandra-kovaleva) [reply hello_world.py]: Maybe you should use smth like this

```suggestion
print("Hello, everyone")
```

**Pull request merged**